src/phoenix/experimental/evals/functions/classify.py

```diff
@@ -628,3 +631,79 @@ def _default_openai_function(
             "required": required,
         },
     }
+
+
+class RunEvalsPayload(TypedDict):
+    evaluator: Evaluator
+    record: Record
+    row_index: RowIndex
+
+
+def run_evals(
+    dataframe: DataFrame,
+    evaluators: List[Evaluator],
```

Comment:
```suggestion
    evaluators: Iterable[Evaluator],
```

Reply:
The implementation currently relies on the input being a list:
- uses `len`
- requires the ability to iterate multiple times over the evaluators